CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## [5.8.1](https://github.com/voxpupuli/beaker/tree/5.8.1) (2024-05-06)
+
+**Fixed bugs:**
+
+- Add extension parameter to parent tmpfile method signature [\#1863](https://github.com/voxpupuli/beaker/pull/1854) ([ekohl](https://github.com/ekohl))
```

Comment:
why? https://github.com/voxpupuli/beaker/pull/1854
looks like the wrong PR is linked